<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaVodka</class>
 <widget class="QMainWindow" name="ventanaVodka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(255, 220, 150, 255), /* Color claro, hacia la izquierda */
        stop:1 rgba(230, 134, 60, 255)  /* Color naranja oscuro, hacia la derecha */
    );

}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tituloVodka">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>961</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-family: &quot;Calibri&quot;;         /* Fuente Calibri */
    font-size: 30pt;                /* Tamaño de 20pt */
    color: #AC1F01;                 /* Color rojo oscuro especificado */
    font-weight: bold;            /* Normal, ajusta a bold si necesitas */
    text-align: left;               /* Alineación izquierda, cámbialo a center si es necesario */
	background: none;

}
</string>
    </property>
    <property name="text">
     <string>Opciones de Vodka disponibles… Seleccione su favorito</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="vBoton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>850</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #e67e22;
    color: white;                /* Texto blanco */
    font-family: Calibri;         /* Fuente Calibri */
    font-size: 22px;              /* Tamaño de la fuente */
    font-weight: bold;  
    border-radius: 20px;          /* Bordes redondeados */
    padding: 10px 20px;           /* Espaciado interno para agrandar el botón */
}

QPushButton:hover {
   background-color: #d35400;     /* Color de fondo al pasar el mouse */
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vDestornillador">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/opciones vodka/Destornillador.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vTonic">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/opciones vodka/Vodka Tonic.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vCaipiroska">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>150</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/opciones vodka/Caipiroska.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vSaltyDog">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/opciones vodka/saltyDog.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vCitrusPunch">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/opciones vodka/citrus vodka punch.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vCitrusBreeze">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>500</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/opciones vodka/vodkaCitrusBreeze.png);
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>vSaltyDog</zorder>
   <zorder>tituloVodka</zorder>
   <zorder>vBoton</zorder>
   <zorder>vDestornillador</zorder>
   <zorder>vTonic</zorder>
   <zorder>vCaipiroska</zorder>
   <zorder>vCitrusPunch</zorder>
   <zorder>vCitrusBreeze</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../Bebidas/bebidasVodka/bebidasVodka.qrc"/>
 </resources>
 <connections/>
</ui>
